news/_posts/2021-12-14-prestashop-beyond-1-7.md

```diff
@@ -0,0 +1,105 @@
+
+Our new release roadmap will adhere more closely to the SemVer definition: a version is considered "major" when it introduces breaking changes, while minor versions only add features in a retrocompatible way. By following this principles strictly, upgrading from a major version to the minor version following it should be as easy as upgrading to a patch version, as everything is expected to work the same way. In this sense, it also means that a major version doesn't necessarily mean _big_ changes, just _some_ changes.
+
+In order to accelerate our delivery cadence, starting on our next release, will begin alternating between major versions, focusing on refactoring and technology advancements, and minor versions, emphasizing on new or improved features.
+
+This means that **the next PrestaShop release will be a major version**. This will be followed by a minor version, then a major version, then a minor version, and so on.
+
+Our target is to be able to publish a release every 6–9 months. Since every other release will be a major version, you should expect a major release every 12 to 18 months.
+
+Now let's talk about support. Currently, patch versions are released at most 6 weeks counting from the moment the maintainers decided that a given bug justifies a patch release. Since we only publish patches for the latest version, this effectively means that support of a given PrestaShop minor version generally ends the moment the following minor version is published (e.g. support for 1.7.7.x version ended when 1.7.8.0 was released). This will continue to be true in our new release roadmap.
+
+However, we understand that upgrading to a major version will be harder than upgrading to a minor or patch version, and that people might choose not to upgrade to a new major version as soon as it's released. Because of this, we decided to provide extended critical and security bug fixes for minor versions, as necessary, for up to 12 months after the following major version has been released.
+
+Here is what our ideal release & support roadmap should look like:
+
+![Ideal roadmap](/assets/images/2021/12/roadmap.png)
+
+To recap:
+
+- A release every 6~9 months
+- Alternating minor and major versions
+- Patches when necessary between releases
+- Up to 1 year of additional critical fixes for minor versions
+
+## Say hello to PrestaShop 8
+
+Our next release, **PrestaShop 8**, will mark the simplification of the project's version numbering scheme. The "1" that preceded previous versions, a heritage that didn't carry any semantic meaning, will no longer be included in future releases.
+
+![New version scheme](/assets/images/2021/12/v8-scheme.png)
+
+
+Adopting the industry standard 3-group version scheme will allow to simplify the way PrestaShop versions are identified, as well as enhance interoperability with third-party software, like Composer, who depend on standardized versions.
```

Comment:
```suggestion
Adopting the industry standard 3-group version scheme will allow to simplify the way PrestaShop versions are identified, as well as enhance interoperability with third-party software, like Composer, who depends on standardized versions.
```

Reply:
"depend" relates to third-party software, not composer (hence the plural)